cocos/platform/CCPosixFileStream.cpp

```diff
@@ -41,7 +41,7 @@ static int pfs_posix_open(const std::string& path, FileStream::Mode mode, PXFile
 
 // posix standard wrappers
 static int pfs_posix_read(PXFileHandle& handle, void* buf, unsigned int size) { return static_cast<int>(posix_read(handle._fd, buf, size)); }
-static off_t pfs_posix_seek(PXFileHandle& handle, off_t offst, int origin) { return posix_lseek(handle._fd, offst, origin); }
```

Comment:
Do you check the off_t on windows, I see the sizeof(off_t) on windows is 32bit

Reply:
```
#ifndef __off_t_defined
# ifndef __USE_FILE_OFFSET64
typedef __off_t off_t;
# else
typedef __off64_t off_t;
# endif
# define __off_t_defined
#endif 
```

Do we want to define `__USE_FILE_OFFSET64`?

Reply:
Ok, do you want me to explicitly set it to int64_t instead of off_t?

Reply:
Yes